tests/scrape.py

```diff
@@ -1143,4 +1167,10 @@ def getnumber(text):
                 continue
     raise ScrapeError('no number found in %r' % text)
 
+def get_all_text(elem):
+    """Returns all texts in the subtree of the element.
```

Comment:
sorry for being inconsistent from the last review, but I actually think that we should call out the type of the element. don't need to say it's etree.Element. "an element returned by xpath() or cssselect()" is probably good enough.